src/util/validator.js

```diff
@@ -70,8 +70,8 @@ module.exports = class Validator {
 
   validateMessage (m) {
     assert(m.ledger, 'must have a ledger')
-    assert(m.data, 'must have data')
-    assertObject(m.data, 'data')
+    assert(m.ilp, 'must have ilp field')
```

Comment:
I don't think `ilp` should be required. Route broadcasting messages have a `custom` field, but not `ilp`.